TwoHoSun/Network/Authenticator.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  Authenticator.swift
+//  TwoHoSun
+//
+//  Created by 235 on 11/13/23.
+//
+import Combine
+import SwiftUI
+
+import Moya
+
+@Observable
+class Authenticator {
+    var authState: TokenState = .none {
+        didSet {
+            authStateSubject.send(authState)
+        }
+    }
+
+    var accessToken: String? {
+        if let accessToken = KeychainManager.shared.readToken(key: "accessToken") {
+            return accessToken
+        }
+        return nil
```

Comment:
어차피 optional이라면 아래와 같이 할 수 있지않을까요?
```suggestion
        KeychainManager.shared.readToken(key: "accessToken") 
```